homework_02112020/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <input id="a" type="number">
+    <input id="b" type="number">
+    <input type="text" id="result" value="" />
+    <button onclick="count()">Result</button>
+    <hr />
+    <div onclick="handleOwnMouseClick(event)" style="height: 100px;width: 100px;background: red;"></div>
```

Comment:
không được phép viết inline style
```suggestion
    <div onclick="handleOwnMouseClick(event)" style="height: 100px; width: 100px; background: red;"></div>
```
Và kể cả vậy thì dấu cách đâu